<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(138, 138, 138);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(145, 145, 145)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="HR_Frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>349</y>
       <width>150</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="HR_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10.5pt &quot;Segoe UI&quot;;
color: rgb(0, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>HR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>3</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(154, 154, 154);
border:2px</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_hr">
      <property name="geometry">
       <rect>
        <x>26</x>
        <y>10</y>
        <width>100</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 0);
font: 24pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="O2_Frame">
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>349</y>
       <width>150</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="O2_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10.5pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>SPO2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_spo2">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>10</y>
        <width>100</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 0);
font: 24pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="NIBP_Frame">
     <property name="geometry">
      <rect>
       <x>453</x>
       <y>349</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="NIBP_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10.5pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>NIBP</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_nibp_L">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>100</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>L</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_nibp_h">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>100</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>H</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_nibp_m">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>100</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>M</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="NIBP_label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>29</y>
        <width>41</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="NIBP_label_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>29</y>
        <width>41</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>(</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="NIBP_label_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>29</y>
        <width>41</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Segoe UI&quot;;
color: rgb(255, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="TMEP_Frame">
     <property name="geometry">
      <rect>
       <x>302</x>
       <y>349</y>
       <width>150</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="TEMP_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10.5pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>TEMP</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_t2">
      <property name="geometry">
       <rect>
        <x>26</x>
        <y>10</y>
        <width>100</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>00.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_e">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>440</y>
       <width>266</width>
       <height>43</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 22pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>EXIT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_s">
     <property name="geometry">
      <rect>
       <x>267</x>
       <y>440</y>
       <width>266</width>
       <height>43</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 22pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="PlotWidget" name="graphWidget_ECG" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>174</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0)</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;Segoe UI&quot;;
color : rgb(0, 255, 0)</string>
      </property>
      <property name="text">
       <string>ECG</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="PlotWidget" name="graphWidget_Pleth" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>175</y>
       <width>800</width>
       <height>173</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 0, 0)</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;Segoe UI&quot;;
color : rgb(255, 255, 0)</string>
      </property>
      <property name="text">
       <string>Pleth</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_save">
     <property name="geometry">
      <rect>
       <x>534</x>
       <y>440</y>
       <width>266</width>
       <height>43</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 22pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>SAVE</string>
     </property>
    </widget>
    <zorder>O2_Frame</zorder>
    <zorder>HR_Frame</zorder>
    <zorder>TMEP_Frame</zorder>
    <zorder>NIBP_Frame</zorder>
    <zorder>pushButton_e</zorder>
    <zorder>pushButton_s</zorder>
    <zorder>graphWidget_ECG</zorder>
    <zorder>graphWidget_Pleth</zorder>
    <zorder>pushButton_save</zorder>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>